controllers/restic_test.go

```diff
@@ -271,10 +271,6 @@ func TestDataMoverRestoreReconciler_buildDMRResticSecret(t *testing.T) {
 					ResticSecretRef: corev1.LocalObjectReference{
 						Name: resticSecret,
 					},
-					DestinationClaimRef: corev1.ObjectReference{
```

Comment:
For `TestDataMoverRestoreReconciler_buildDMRResticSecret` func: Should we remove dmb references as well?